risc0/r0vm/src/lib.rs

```diff
@@ -112,10 +112,6 @@ pub fn main() {
         .init();
 
     let args = Cli::parse();
-    if let Some(port) = args.mode.port {
-        run_server(port);
-        return;
-    }
 
     if args.id {
```

Comment:
Turns out this isn't the case. If you do `--id` with the existing code, you get:
```
error: the following required arguments were not provided:
  <--port <PORT>|--elf <ELF>|--image <IMAGE>>
```

This is because `Mode` is already marked as required.

I tried adding `requires = "elf"` but the behavior didn't change.